src/media/css/tiles.styl

```diff
@@ -71,6 +71,10 @@ $greater-than-mobile = unquote('(min-width: 330px)');
         margin: 0 auto;
         width: 50%;
     }
+    +website-detail--wide() {
```

Comment:
I'm unsure of how we should handle the two detail types. We had discussed creating a generic `.app-list-tile` for the listing pages, perhaps we should do the same here? Then we'd have a block of common styles followed by any specific app/website styles. Is that a good idea or will this be more of a maintenance issue down the road than app default with website overrides?

```stylus
+detail--wide() {
    display: flex;
    justify-content: center;
    padding-bottom: 50px;
    position: relative;
    margin: 0 auto;
}
+app-detail--wide() {
    height: 240px;
    width: 50%;
}
+website-detail--wide() {
    width: 75%;
}
```